src/QueryBuilder.php

```diff
@@ -101,6 +101,13 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
         return parent::simplePaginate($perPage, $columns, $pageName, $page);
     }
 
+    public function chunk($count, callable $callback)
+    {
+        $this->parseSorts();
+
```

Comment:
i've removed the `addAllRequestedFields` block because chunk calls to get method and aplly this requested fields on his function